File: AEDS/EXERCÍCIOS/NIVELAMENTOb/exercicio1.c
#include <stdlib.h>
#include <stdio.h>

int comparar(int vet[], int N, int x) {
    for (int i = 0; i < N; i++) {
        if (vet[i] == x) return 1;
    }
    return 0;
}

int main () {
    int N; scanf("%d", &N);
    int vet[N]; for (int i = 0; i < N; i++) scanf("%d", &vet[i]);
    int x; scanf("%d", &x);
    int comp = comparar(vet, N, x);
    if (comp == 1) printf("SIM\n");
    else printf("NAO\n");
}

File: AEDS/EXERCÍCIOS/NIVELAMENTO1/exercicio1.java
import java.util.Scanner;

public class exercicio1 {
    public static int comparar(int vet[], int N, int x) {
        for (int i = 0; i < N; i++) {
            if (vet[i] == x)
                return 1;
        }
        return 0;
    }
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int vet[] = new int[N]; for (int i = 0; i < N; i++) vet[i] = sc.nextInt();
        int x = sc.nextInt();
        int comp = comparar(vet, N, x);
        if (comp == 1) System.out.println("SIM");
        else System.out.println("NAO");
        sc.close();
    }
}
